spec/system/games_index_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+RSpec.describe 'GamesIndex', type: :system, js: true do
+  include ActiveJob::TestHelper
+  let!(:admin) { FactoryBot.create(:admin) }
+  before do
+    FactoryBot.create(:kingdom_hearts, user_id: admin.id)
+    FactoryBot.create(:user)
+    9.times do
+      FactoryBot.create(:game, user_id: admin.id)
+      FactoryBot.create(:user)
+    end
+    id = Game.first.id
+    user_id = User.first.id
+    for i in 0..9 do
```

Comment:
テストなんで気にしなくてもいいんんですけど、eachでも書けます。

```suggestion
    (0..9).each do |i|
```

Reply:
case文と共に修正しました。